src/docfx/config/Config.cs

```diff
@@ -150,7 +150,7 @@ private static Config LoadCore(string docsetPath, string configPath, CommandLine
             try
             {
                 var optionConfigObject = options?.ToJObject();
-                var configObject = ExpandAndNormalize(JsonUtility.Merge(LoadConfigObject(docsetPath, configPath, extend, restoreMap), optionConfigObject));
```

Comment:
you lost the function to pass the extends from cli?

Reply:
we do not support extend in cli

---
In reply to: [210170784](https://github.com/dotnet/docfx/pull/3196#discussion_r210170784) [](ancestors = 210170784)

Reply:
we can support it in the feature but if you change that we will never support it.

---
In reply to: [210460133](https://github.com/dotnet/docfx/pull/3196#discussion_r210460133) [](ancestors = 210460133,210170784)